src/lab4.py

```diff
@@ -0,0 +1,49 @@
+import textwrap
+
+class laptop:
+    def __init__(self,  developer = "", frequency = 0.0, ram = 0):
+
+        self.__developer = developer
```

Comment:
Rename please this variable because you to put in the name of the manufacturer

Reply:
done